ui/plugins/eu.esdihumboldt.hale.ui.functions.numeric/src/eu/esdihumboldt/hale/ui/functions/numeric/SequentialIDParameterPage.java

```diff
@@ -377,13 +377,22 @@ protected String generateExample() {
 		String suffix = (this.suffix == null) ? ("") : (this.suffix.getText());
 		String startValue = (this.startValue == null) ? ("") : (this.startValue.getText());
 
+		String specialCharactersString = "!@#$%&*()'+,-./:;<=>?[]^_`{|}";
```

Comment:
@Flaminietta The method `generateExample()` doesn't seem to be the right place to check the validity of the user input. A better place to do that would be the `ModifyListener` of the input field (or a method called from there).